src/main.ts

```diff
@@ -1,7 +1,13 @@
+import { resolve } from 'node:path'
+import process from 'node:process'
+
 import { defineCommand } from 'citty'
 import { provider } from 'std-env'
+import { x } from 'tinyexec'
```

Comment:
This import seems could be lazy-loaded to avoid runtime eval overhead unless we hit the fallback